invenio_pidstore/config.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think you likely need to add this to the documentation as well similar to:

https://github.com/inveniosoftware/invenio-rest/blob/master/docs/configuration.rst
https://github.com/inveniosoftware/invenio-rest/blob/master/docs/index.rst